.github/CODEOWNERS

```diff
@@ -0,0 +1,26 @@
+#########################
+#####  Global Rule  #####
+#########################
+*                                                       @hashgraph/dev-experience
+#########################
+#####  Core Files  ######
+#########################
+
+# NOTE: Must be placed last to ensure enforcement over all other rules
+
+# Protection Rules for Github Configuration Files and Actions Workflows
+/.github/                                               @hashgraph/devops-ci @hashgraph/release-engineering-managers
+
+# Self-protection for root CODEOWNERS files (this file should not exist and should definitely require approval)
+/CODEOWNERS                                             @hashgraph/release-engineering-managers
+
+# Protect the repository root files
+/README.md                                              @hashgraph/devops-ci @hashgraph/release-engineering-managers @hashgraph/dev-experience
+**/LICENSE                                              @hashgraph/release-engineering-managers
+
+# CodeCov configuration
+**/codecov.yml                                          @hashgraph/devops-ci @hashgraph/release-engineering-managers
```

Comment:
This file does not exist?

Reply:
I want this in here for the future, but yes at this time it does not exist.